new/LIT7033Yelebbesu.xml

```diff
@@ -0,0 +1,80 @@
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7033Yelebbesu" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ይለብሱ፡ አልባሰ፡ ፀርቅ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Yǝlabbǝsu ʾalbasā ḍarq ... (Qǝne of śǝllāse type)</title>
+                <title xml:lang="en" corresp="#t1">They wear clothes of rags ...</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>This <term key="Qene">qǝne</term> poem attested in <ref type="mss" corresp="Tanasee6"/>, 
+                    readable only in part, is marked as being of the type <foreign xml:lang="gez">śǝllāse</foreign>. 
+                    It is attributed to <persName ref="PRS14503GabraMarQene">mamhǝr Gabra Māryām of Kǝbrān</persName>.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Sellase"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <t:change xmlns:t="http://www.tei-c.org/ns/1.0" who="DN" when="2024-07-17">Created entity</t:change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7033Yelebbesu" passive="Tanasee6"/>
+                    <relation name="saws:isAttributedToAuthor" active="LIT7033Yelebbesu" passive="PRS14503GabraMarQene"/>
+                </listRelation>
+                <div type="edition">
```

Comment:
Why is `<div type="edition">` nested in `<div type="bibliography">`? I think, it should be parallel.

```
<div type="bibliography">
                <listRelation>
                    <relation name="lawd:hasAttestation" active="LIT7033Yelebbesu" passive="Tanasee6"/>
                    <relation name="saws:isAttributedToAuthor" active="LIT7033Yelebbesu" passive="PRS14503GabraMarQene"/>
                </listRelation>
</div>
 <div type="textpart" xml:lang="gez">
                        <note>The text is taken from <ref type="mss" corresp="Tanasee6"/>
                            (signs indicating division into verses are poorly visible).</note>
                        <ab> 
                            ሥላሴ፡ ዘ<persName ref="PRS14503GabraMarQene">መምህር፡ ገብረ፡ ማርያም፡</persName> ዘ<placeName ref="INS0357Kebran">ደሴተ፡ ክብራን፡</placeName>                            
                            <l n="1">ይለብሱ፡ አልባሰ፡ ፀርቅ፡ ነዳያነ፡ ሕግ፡ መቅደንዮስ፡ ወሰባልዮስ፡ አኃው፡</l>
                            <l n="2"><supplied reason="undefined" resp="DN">ጸ</supplied>አቱ፡ ልብሰ፡ ሜላተ፡ ወርቅ፡ ቦሙ፤</l> 
                                <l n="3">ለደቀስዮስ፡ ወኒቆላዎስ፡ ማርያም፡ እሞሙ፤</l>
                            <l n="4">ልብሰ፡ ሜላተ፡ ወርቅ፡ ጽሩይ፡ አምጣነ፡ ትፈትል፡ ሎሙ፤</l>
                            <l n="5">ወዜና፡ ወልደ፡ አብ፡ አምላኮሙ፡ ዜናዊ፡ ገብርኤል፡ <supplied reason="undefined" resp="DN">ሶበ፡ አስም</supplied>ዖሙ፤</l>
                            <l n="6">በትፍሥሕት፡ ወሐሤት፡ ቆሙ።</l>                          
                        </ab>
                    </div>
```